metricbeat/docs/gettingstarted.asciidoc

```diff
@@ -122,100 +122,98 @@ endif::[]
 
 . Extract the contents of the zip file into `C:\Program Files`.
 
-. Rename the `metricbeat-<version>-windows` directory to `Metricbeat`.
+. Rename the +{beatname_lc}-<version>-windows+` directory to +{beatname_uc}+.
 
 . Open a PowerShell prompt as an Administrator (right-click the PowerShell icon
 and select *Run As Administrator*). If you are running Windows XP, you may need
 to download and install PowerShell.
 
-. From the PowerShell prompt, run the following commands to install Metricbeat
+. From the PowerShell prompt, run the following commands to install {beatname_uc}
 as a Windows service:
 +
-[source,shell]
+["source","sh",subs="attributes"]
 ----------------------------------------------------------------------
-PS > cd 'C:\Program Files\Metricbeat'
-PS C:\Program Files\Metricbeat> .\install-service-metricbeat.ps1
+PS > cd 'C:{backslash}Program Files{backslash}{beatname_uc}'
+PS C:{backslash}Program Files{backslash}{beatname_uc}> .{backslash}install-service-{beatname_lc}.ps1
 ----------------------------------------------------------------------
 
 NOTE: If script execution is disabled on your system, you need to set the
 execution policy for the current session to allow the script to run. For
-example: `PowerShell.exe -ExecutionPolicy UnRestricted -File
-.\install-service-metricbeat.ps1`.
+example: +PowerShell.exe -ExecutionPolicy UnRestricted -File
+.{backslash}install-service-{beatname_lc}.ps1+.
 
 endif::[]
 
-Before starting Metricbeat, you should look at the configuration options in the
-configuration file, for example `C:\Program Files\Metricbeat\metricbeat.yml`.
+Before starting {beatname_uc}, you should look at the configuration options in the
+configuration file, for example +C:{backslash}Program Files{backslash}{beatname_uc}{backslash}{beatname_lc}.yml+.
 For more information about these options, see
-<<configuring-howto-metricbeat>>.
+<<configuring-howto-{beatname_lc}>>.
 
-[[metricbeat-configuration]]
-=== Step 2: Configure Metricbeat
+[id="{beatname_lc}-configuration"]
+=== Step 2: Configure {beatname_uc}
 
 include::../../libbeat/docs/shared-configuring.asciidoc[]
 
-Metricbeat uses <<metricbeat-modules,modules>> to collect metrics. You configure
-each module individually. The following example shows the default configuration
-in the `metricbeat.yml` file. The system status module is enabled by default to
-collect metrics about your server, such as CPU usage, memory usage, network IO
-metrics, and process statistics:
-
-[source, shell]
--------------------------------------
-metricbeat.modules:
-- module: system
-  metricsets:
-    - cpu
-    - filesystem
-    - memory
-    - network
-    - process
-  enabled: true
-  period: 10s
-  processes: ['.*']
-  cpu_ticks: false
--------------------------------------
-
-The following example shows how to configure two modules: the system module
-and the Apache HTTPD module:
-
-[source, shell]
--------------------------------------
```

Comment:
I wonder why your removed this part? I remember I added it because in the early days of Metricbeat on discuss several times the question popped up on how to configure 2 modules or if it's possible to configure the same module twice. I was hoping this example here helps.

Reply:
@ruflin I removed it because I wanted the getting started guide to focus using the configs in the modules.d directory because using the enable modules command is a better getting started experience. I thought users would look at this topic for config metricbeat.yml config examples: https://www.elastic.co/guide/en/beats/metricbeat/master/configuration-metricbeat.html

Do you want me to add the deleted example to the ones shown in configuration-metricbeat.html, or do you think the example that are there are good enough?

Reply:
@dedemorton You are right, it's all there. Forgot that we also have there some good examples. +1 on removing it here.